modules/fedimint-mint-client/src/lib.rs

```diff
@@ -858,6 +858,17 @@ impl ClientModule for MintClientModule {
                 }
                 "spend_notes" => {
                     let req: SpendNotesRequest = serde_json::from_value(request)?;
+                    let result = self.spend_notes_with_selector(
+                        &SelectNotesWithExactAmount,
+                        req.amount,
+                        req.try_cancel_after,
+                        req.include_invite,
+                        req.extra_meta
+                    ).await?;
+                    yield serde_json::to_value(result)?;
+                }
+                "spend_notes_expert" => {
+                    let req: SpendNotesExpertRequest = serde_json::from_value(request)?;
                     let result = self.spend_notes(req.min_amount, req.try_cancel_after, req.include_invite, req.extra_meta).await?;
```

Comment:
should we change the default in self.spend_notes?

Reply:
good point!

Reply:
Thinking about it more, changing the behavior would be a bad move. Integrators might depend on the semantics. So instead we can a) remove it or b) deprecate it and remove it later. Deprecation is the right move for now imo.